sceptre/config/reader.py

```diff
@@ -436,7 +437,14 @@ def _render(self, directory_path, basename, stack_group_config):
                 stack_group_config.get("j2_environment", {}),
             )
             j2_environment = Environment(**j2_environment_config)
-            template = j2_environment.get_template(basename)
+
+            try:
+                template = j2_environment.get_template(basename)
+            except Exception as err:
+                raise SceptreException(
+                    f"{Path(directory_path, basename).as_posix()} - {err}"
+                )
```

Comment:
Same comment I made on the other PR in #1269 
```suggestion
                ) from err
```